src/workingtitle-vcockpits-instruments-airliners/html_ui/Pages/VCockpit/Instruments/Airliners/Shared/WT/AltimeterIndicator.js

```diff
@@ -1551,39 +1602,59 @@ class Jet_PFD_AltimeterIndicator extends HTMLElement {
     updateBaroPressure(_mode) {
         let baroPreset = SimVar.GetSimVarValue("L:XMLVAR_Baro1_SavedPressure", "number") / 16;
         
-        if (this.pressureSVG) {
+        if (this.pressureSVGLeftPart) {

```

Comment:
@flymypi Again, not sure what to test for.

Reply:
This check prevents the update loop tripping over a not yet constructed or "damaged" SVG structure. Because the following code depends on all three elements you should check for them.

```suggestion
        if (this.pressureSVGLeftPart && this.pressureSVGCenterPart && this.pressureSVGRightPart) {
```